New Item Extension mod element (#2479)
MCreator/MCreator: plugins/mcreator-localization/help/default/item_extension/item.md

> This is the target item of this extension.

Reviewer: Определяет, какой предмет будет обладать данными свойствами.

> This is the item that will receive the properties.

Reviewer: I don't think properties can be received (at least never heard that).
```suggestion
This is the target item of this extension.
```
